content/applications/websites/ecommerce/maximizing_revenue/cross_selling.rst

```diff
@@ -1,55 +1,68 @@
+As an example, you could suggest a mouse or a keyboard when someone buys a computer.
 
-.. note::
-    There is an algorithm to figure out the best accessories to display in case
-    several items are added to cart. If any item is the accessory of several products
-    added to cart, it is most likely that it will be atop the list of suggested accessories.
+To add *Accessory Products*, go the :menuselection:`Website --> Products --> Products` and, under
+the :guilabel:`Sales Tab`, add your products to the :guilabel:`Accessory Products` field.
 
-Optional products when adding to cart
-=====================================
+    .. image:: cross_selling/accessory-product.png
+       :align: center
+       :alt: Add an accessory product to product form.
 
-Optional products are directly related to the item added to cart (e.g. for computers:
-warranty, OS software, extra components). Whenever the main product is added to cart,
-such a new screen pops up as an extra step.
+Now, go to your website and add the product to your cart. On the payment step, you can see your
+accessory products suggested to the customer.
 
-.. image:: cross_selling/cross_sell_optional_web.png
-   :align: center
+    .. image:: cross_selling/accessory-products-customer-view.png
+       :align: center
+       :alt: See of payment view and accessory product suggestion.
 
-To publish optional products:
+.. seealso::
+   `Alternative, Accessory, and Optional Products
+   <https://www.odoo.com/slides/slide/alternative-accessory-and-optional-products-637>`_.
 
-* Install *eCommerce Optional Products* addon in *Apps* menu. Remove the default filter to
-  search on addons as well, otherwise only main apps show up.
+.. _cross_selling/optional:
 
-.. image:: cross_selling/cross_sell_optional_install.png
-   :align: center
+Optional Products
+=================
 
-* Select optional items from the *Sales* tab of the product detail form.
+**Optional products** are products directly related to an item. Each time the product is added to a
+cart, a pop-up appears, suggesting other products to buy.
 
-.. image:: cross_selling/cross_sell_optional.png
-   :align: center
+To enable optional products, go to :menuselection:`Website --> Configuration --> Settings` and tick
+:guilabel:`Optional Products`. Then, go to :menuselection:`Website --> Products --> Products` and
+open the item you want to add an optional product to. In the :guilabel:`Sales tab` of that product,
+add as many optional products as you want.
 
-.. note::
-    The quantity of optional items added to cart is the same than the main item.
+    .. image:: cross_selling/optional-product.png
+       :align: center
+       :alt: Add an optional product to product form.
 
+From now, when a customer adds this product to their cart, a pop-up appears and suggests to also add
+the optional product(s).
+
+    .. image:: cross_selling/cross-selling-pop-up.png
```

Comment:
"cross-selling" is already in the folder's name. Not a mistake per se, but it can be named "pop-up" imo.